.github/workflows/publish-techdocs.md

```diff
@@ -0,0 +1,41 @@
+# Reusable workflow: Publish techdocs
+
+This workflow helps you building your project's documentation and publishing it to [EngHub](https://enghub.grafana-ops.net).
+Please keep in mind that for this you also need to first register your repository with EngHub.
+You can find details on this [here](https://enghub.grafana-ops.net/docs/default/component/enghub/user-guides/add-gh-repo/).
+
+## Usage example
+
+```yaml
+name: Publish TechDocs
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+      - 'mkdocs.yml'
+      - 'catalog-info.yaml'
+      - '.github/workflows/publish-docs.yml'
+concurrency:
+  group: '${{ github.workflow }}-${{ github.ref }}'
+  cancel-in-progress: true
+jobs:
+  publish-docs:
+    uses: grafana/shared-workflows/.github/workflows/publish-techdocs.yaml@main
+    secrets: inherit
+    with:
+      namespace: default
+      kind: component
+      name: COMPONENT_NAME
+```
+
+## Inputs
+
+| Name | Type | Description |
+|-|-|-|
+| `namespace` | string | The entities namespace (usually `default`) |
```

Comment:
```suggestion
| `namespace` | string | The entity's namespace (usually `default`) |
```

also would it be worth clarifying this is the _catalog_ namespace and not the Kubernetes one?

Reply:
Done :)